api_end_end_testing/src/test/java/tds/user/api/tests/CreateUserTest.java

```diff
@@ -0,0 +1,247 @@
+package tds.user.api.tests;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.jayway.restassured.RestAssured.*;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Header;
+import com.jayway.restassured.http.ContentType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.testng.annotations.Test;
+import tds.user.api.model.UserInfo;
+import tds.user.api.model.RoleAssociation;
+
+/*
+    Authenticate user
+    Testing HTTP POST of https://sso-deployment.sbtds.org/auth/oauth2/access_token?realm=/sbac
+    and success status 200
+ */
+public class CreateUserTest extends BaseUri{
+    String accessToken = null;
+    String email = null;
+
+    @Test
+    public void getAuthenticationCode() {
```

Comment:
I'm not sure this feels like a test to me since we are not testing the OAuth piece but really just need the token back.  I would suggest moving this to part of the @BeforeClass method that you have and getting the access_token there.  Then each Test will have access to it since it is the base class.